app-info/metainfo/org.gnome.Software.WebApp_02310f2fd700a63d17ecc2d4b261f6956d8a0193.metainfo.xml.in

```diff
@@ -13,6 +13,20 @@
   <metadata_license>FSFAP</metadata_license>
   <developer_name>Flathub</developer_name>
   <icon type="remote" width="512" height="512">https://discourse-flathub-org-uploads.s3.dualstack.us-east-1.amazonaws.com/optimized/1X/be4563846333bda6607c6c3c5b226dff9d213866_2_512x512.png</icon>
+  <screenshots>
+    <screenshot>
```

Comment:
The primary screenshot (usually the first) should have the attribute `type="default"`.